src/testUtils.js

```diff
@@ -20,14 +22,20 @@ export const formatMessage = (msg, values) => {
  * @return {func} - mock component with nested children.
  *
  * usage:
- *   mockNestedComponent('Card', { Body: 'Card.Body', ... });
+ *   mockNestedComponent('Card', { Body: 'Card.Body', Form: { Control: { Feedback: 'Form.Control.Feedback' }}... });
  *   mockNestedComponent('IconButton', 'IconButton');
  */
 export const mockNestedComponent = (name, contents) => {
-  if (typeof contents !== 'object') { return contents; }
+  if (typeof contents !== 'object') {
+    return contents;
+  }
   const fn = () => name;
   Object.defineProperty(fn, 'name', { value: name });
-  Object.keys(contents).forEach(nestedName => { fn[nestedName] = contents[nestedName]; });
+  Object.keys(contents).forEach((nestedName) => {
+    const value = contents[nestedName];
+    fn[nestedName] = typeof value !== 'object' ? value
```

Comment:
If the "else"(:) is split onto a new line, I prefer the "then"(?) to be as well for readability.
```suggestion
    fn[nestedName] = typeof value !== 'object'
      ? value
```